easybuild/easyconfigs/o/openkim-models/openkim-models-20210811-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'CMakeMake'
+
+name = 'openkim-models'
+version = '20210811'
+
+homepage = 'https://openkim.org/'
+description = """Open Knowledgebase of Interatomic Models.
+
+OpenKIM is an API and a collection of interatomic models (potentials) for
+atomistic simulations.  It is a library that can be used by simulation programs
+to get access to the models in the OpenKIM database.
+
+This EasyBuild installs the models.  The API itself is in the kim-api
+package.
+ """
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkg-config', '0.29.2'),
```

Comment:
Please try with `pkgconf` instead, which is the (almost always) drop in replacement (pkg-config isn't maintained anymore which is why we didn't have it for GCC 12.3.0  yet)

Reply:
Good point.  Yes, it looks like it just works with pkgconf, so I'll use that from now on.  Thanks for pointing this out, @Micket !

Reply:
I have also amended the ASAP documentation on readthedocs to mention the pkgconf package instead of pkg-config.